src/main/java/com/minecolonies/coremod/entity/pathfinding/Pathfinding.java

```diff
@@ -91,153 +90,156 @@ public static void debugDraw(final double frame)
         {
             for (@NotNull final Node n : debugNodesNotVisited)
             {
-                debugDrawNode(n, (byte) 255, (byte) 0, (byte) 0);
+                debugDrawNode(n, 1.0F, 0F, 0F);
             }
 
             for (@NotNull final Node n : debugNodesVisited)
             {
-                debugDrawNode(n, (byte) 0, (byte) 0, (byte) 255);
+                debugDrawNode(n, 0F, 0F, 1.0F);
             }
 
-            if (debugNodesPath != null)
+            for (@NotNull final Node n : debugNodesPath)
             {
-                for (@NotNull final Node n : debugNodesPath)
-                {
-                    debugDrawNode(n, (byte) 0, (byte) 255, (byte) 0);
-                }
+                debugDrawNode(n, 0F, 1.0F, 0F);
             }
         }
         catch (final ConcurrentModificationException exc)
         {
             Log.getLogger().catching(exc);
         }
 
-        GL11.glPopAttrib();
-        GL11.glPopMatrix();
+        GlStateManager.popAttrib();
+        GlStateManager.popMatrix();
     }
 
     @SideOnly(Side.CLIENT)
-    private static void debugDrawNode(@NotNull final Node n, final byte r, final byte g, final byte b)
+    private static void debugDrawNode(@NotNull final Node n, final float r, final float g, final float b)
     {
-        GL11.glPushMatrix();
-        GL11.glTranslated((double) n.pos.getX() + 0.375, (double) n.pos.getY() + 0.375, (double) n.pos.getZ() + 0.375);
-
-        final float f = 1.6F;
-        final float f1 = (float) (0.016666668D * f / 2);
-
-        //  Nameplate
+        GlStateManager.pushMatrix();
+        GlStateManager.translate((double) n.pos.getX() + 0.375, (double) n.pos.getY() + 0.375, (double) n.pos.getZ() + 0.375);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 0.375 to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 0.375 to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 0.375 to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)